package.json

```diff
@@ -67,6 +67,7 @@
     "grunt-contrib-clean": "^0.6.0",
     "grunt-contrib-watch": "^0.6.1",
     "grunt-simple-mocha": "^0.4.0",
+    "haikro": "^2.3.2",
```

Comment:
as a workaround you could drop it into a `Makefile` with https://github.com/Financial-Times/n-makefile/blob/master/Makefile#L9 to force it to execute a bash shell even if you, as a developer, are using zsh.